### PR TITLE
Added code comments to quickstart code

### PR DESCRIPTION
**Pull request by NeoKish**
Signed-off-by: Kishan Savant

Missing code comments were added based on the README.md

### CODE REVIEW

File: docs/quick.rst [hunk 1]
```diff
@@ -53,19 +53,25 @@ Just the code
     >>> import pandas as pd
     >>> import nannyml as nml
     >>> from IPython.display import display
+
+    >>> # Load dummy data
```
Comment (nnansters) [inline docs/quick.rst:57]: Maybe not dummy data but synthetic data?

Comment (NeoKish) [reply docs/quick.rst]: Done

**Comment by nnansters** [comment]
Nice! Thanks for your contribution!

**Comment by NeoKish** [comment]
Thank you for the review

**Pull request merged**